spec/support/vcr_setup.rb

```diff
@@ -7,8 +7,10 @@
   config.hook_into :webmock
   config.ignore_localhost = true
   config.configure_rspec_metadata!
-  config.filter_sensitive_data('<HIDDEN-STRIPE_SECRET_TEST_API_KEY>') { ENV.fetch('STRIPE_SECRET_TEST_API_KEY', nil) }
-  config.filter_sensitive_data('<HIDDEN-STRIPE_CUSTOMER>') { ENV.fetch('STRIPE_CUSTOMER', nil) }
-  config.filter_sensitive_data('<HIDDEN-STRIPE_ACCOUNT>') { ENV.fetch('STRIPE_ACCOUNT', nil) }
   config.ignore_hosts('localhost', '127.0.0.1', '0.0.0.0', 'api.knapsackpro.com')
+
+  # Filter sensitive environment variables
+  ['STRIPE_SECRET_TEST_API_KEY', 'STRIPE_CUSTOMER', 'STRIPE_ACCOUNT'].each do |env_var|
+    config.filter_sensitive_data("<HIDDEN-#{env_var}>") { ENV.fetch(env_var, nil) }
+  end
 end
```

Comment:
Extra smile for that commit message :-D

Reply:
Oh, but it looks like the line is too long... And rubocop complains. Other than that, looks great to me!

Reply:
🤦  Oops, fixed now.